connector/tableau/looker-jdbc/connectionFields.xml

```diff
@@ -19,7 +19,7 @@
 <!-- Connection fields, part of the Tableau connector SDK; see manifest.xml. -->
 <connection-fields>
   <field name="server" label="Server" value-type="string" category="endpoint" default-value="localhost">
-    <validation-rule reg-exp="^(([a-zA-Z]|[a-zA-Z][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z]|[A-Za-z][A-Za-z0-9\-]*[A-Za-z0-9])$"/>
+    <validation-rule reg-exp="^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\-]*[A-Za-z0-9])$"/>
```

Comment:
Might as well simplify to `[a-zA-Z0-9]*` right?

Reply:
oh and special chars `.` `\-`

Reply:
What about `^[a-zA-Z0-9\.\-]*$`?

Reply:
<img width="1443" alt="Screenshot 2023-11-08 at 9 40 36 AM" src="https://github.com/looker-open-source/calcite/assets/116591231/e458e09e-0fc1-42a5-9523-52e2254db754">

Reply:
My very scientific tests seem to be passing